### PR TITLE
fix: removed explicit configuration for Dynaconf. 

### PR DESCRIPTION
**Pull request by jstavel**
I have removed explicit configuration for dynaconf.
I have updated README.md by a new section 'Configuration'.
There is an example how to configure Dynaconf in the section.

### CODE REVIEW

**Comment by jstavel** [comment]
The explicit arguments in Dynacon() take precedence over env vars - I guess. That's why I have remove the arguments. And let anyone specify the main config properties over env vars only.

File: README.md [hunk 1]
```diff
@@ -27,6 +27,52 @@ and some helper bits:
 - `insights_client` -- `insights-client`
 - `rhc` -- `rhc`
 
+## Configuration
+
+There is a fixture =test_config= in the defined fixtures.
```
Comment (ptoscano) [inline README.md:32]: style: use `` `...` `` rather than `=...=`

**Review by ptoscano — CHANGES_REQUESTED** [review]

File: pytest_client_tools/test_config.py [hunk 2]
```diff
@@ -6,10 +6,7 @@
 
 class TestConfig:
     def __init__(self):
-        self._settings = Dynaconf(
-            envvar_prefix="PYTEST_CLIENT_TOOLS",
-            settings_files=["settings.toml", ".secrets.toml"],
```
[the remark this answers is not in the document]
Comment (jstavel) [reply pytest_client_tools/test_config.py:11]: I was not able to specify additional file (.secrets.yml) when settings_file arg is presented. .secrets.yml fits better (IMO) to a structured data.

File: pytest_client_tools/test_config.py [hunk 3]
```diff
@@ -6,10 +6,7 @@
 
 class TestConfig:
     def __init__(self):
-        self._settings = Dynaconf(
-            envvar_prefix="PYTEST_CLIENT_TOOLS",
```
[the remark this answers is not in the document]
Comment (jstavel) [reply pytest_client_tools/test_config.py:10]: no problem to use envvar_prefix in the constructor. it makes sense - since it is pytest-client-tools

**Comment by jstavel** [comment]
This PR is duplicated by another one in a PR queue.

**Pull request closed without merging**